### PR TITLE
Try to add style ci

### PR DESCRIPTION
**Pull request by iamashwin99**
Taking inspiration form style checks performed at https://gitlab.gwdg.de/mpsd-cs/mpsd-hpc-tools, this MR tries to implement style checks using `isort`, and `black` using the standard values accepted in spack upstream.

using details menationed in https://github.com/spack/spack/blob/75bf6d665c7a7fe14cd6a0c8e4e3f72457556edb/lib/spack/spack/cmd/style.py and lining configurations present in https://github.com/spack/spack/blob/develop/pyproject.toml

### CODE REVIEW

**Comment by iamashwin99** [comment]
CI for [9604c92](https://github.com/fangohr/octopus-in-spack/pull/85/commits/9604c927f9a04fb8f926ddaf1df38f53a190f206) 
shows that the current package needs style changes, testing with upstream

**Comment by iamashwin99** [comment]
If you like the changes @fangohr I will remove the commits to update the package.py and reenable the other CIs.

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,85 @@
+# Using template from https://github.com/spack/spack/blob/develop/pyproject.toml to get the settings for black and isort
```
Comment (fangohr) [inline pyproject.toml:1]: Good include the source of settings here.

**Comment by fangohr** [comment]
Undecided: I think it is mostly a good idea. A problem is that we need to update the style definitions if they change upstream (in spack). But perhaps that's better than having the wrong spack style before we submit merge requests upstream to spack/spack. Having a separate workflow is good: it helps us identify what is wrong. (And I got the style wrong before, so the check is useful.)

So yes, I'd be happy to include that. (Once the package changes are undone.)

**Pull request merged**